README.md

```diff
@@ -8,7 +8,89 @@ and provide them to systemd services.
 [The module](./modules/vault-secrets.nix) declares a `vault-secrets` option
 which can be used to fetch secrets from vault.
 
-<!-- FIXME: add more documentation -->
+Each service is expected to have a separate "secret" with its own AppRole used
+to log in. Secrets are never written to disk, but kept in tmpfs. By default,
+they are only accessible by `root` and `chmod 400`.
```

Comment:
I think we don't clear write bit for secrets, so it's `600`

Reply:
Right, it runs with `umask 0077`.